005-trim/trim.sh

```diff
@@ -13,6 +13,7 @@ fastq2=$dataDir/$task2.fastq.gz
 log=$logDir/$task.log
 out=$task.trim.fastq.gz
 out2=$task2.trim.fastq.gz
+out3=$task.singleton.fastq.gz
```

Comment:
It would be better to call this `singletons` as you do in the flash script.

Reply:
done